src/Invocation/StaticInvocation.php

```diff
@@ -135,6 +135,10 @@ public function isReturnTypeNullable(): bool
      */
     public function generateReturnValue()
     {
+        if (\strtolower($this->getMethodName()) === '__tostring') {
+            return '';
+        }
+
```

Comment:
Can you please send a separate pull request just for this change? Thanks!

Reply:
Sure thing

Reply:
Done, see #407